File: sentimental-mario-less/mario.py
from cs50 import get_int  # import statements

height = get_int("Height: ")

# input check
while (height < 1 or height > 8):
    height = get_int("Height: ")

# conditionals
if height > 0 and height < 9:
    for i in range(height):
        for j in range(height):
            if (i+j+1) < height:
                print(" ", end="")
            else:
                print("#", end="")
        print()

File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
    int n;
    do{
        n=get_int("Height: ");
    }
    while(n<1);

    for(int i=1; i<=n;i++)
    {
        for(int j=1; j<=n; j++){
            if(j+i>n)
                printf("#");
            else
                printf(" ");
        }
        printf("\n");
    }
}
